src/api-service/__app__/onefuzzlib/azure/nsg.py

```diff
@@ -95,6 +101,10 @@ def update_nsg(nsg: NetworkSecurityGroup) -> Union[None, Error]:
     return None
 
 
+def ok_to_delete(regions: Set[Region], nsg_location: str, nsg_name: str) -> bool:
+    return nsg_location not in regions and nsg_location == nsg_name
```

Comment:
I would rename these to give better context 

```suggestion
def ok_to_delete(active_regions: Set[Region], nsg_region: str, nsg_name: str) -> bool:
    return nsg_region not in active_regions and nsg_region == nsg_name
```